packages/api/src/utils/index.ts

```diff
@@ -50,7 +50,15 @@ export function normalizeNetworkConfig (
         label: value.name
       }
     })
-  )
+    const testnetNetworks = networks.filter(
+      network => !network.label.includes('Mainnet')
+    )
+    const mainnetNetworks = networks.filter(network =>
+      network.label.includes('Mainnet')
+    )
```

Comment:
This is totally the perfect place to give a try to the nice `Array.prototype.groupBy()` method which implements partition logic :wink: 

```ts
const {mainnet, testnet} = networks
  .groupBy((network) => network.label.includes('Mainnet') ? 'mainnet' : 'testnet')
```

The snippet above (warning: untested!) should be more efficient because it iterates over the networks only once :rocket:

Reply:
Firefox is already in 2030 :robot:  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/groupBy#browser_compatibility